wordpress-mobile/WordPress-Android
# Add UI tests for critical login flows

Part of #9296 

This PR adds tests for 

- Log in to WordPress.com using site address option.
- Log in to a self-hosted site using the site address option.
- Start the login flow with the magic link option (ending with the step to request the link). We can revisit options for continuing this test once our planned network mocking is in place.

The PR also updates `WPLoginInputRow` to have a method to set the content description and also sets the content description for the username and password field for the site address login because ID wasn't finding the element. 

To test:

- Start up an emulator, either phone or tablet
- Run the login tests

Ensure

- Code is good
- Tests pass
